tests/test_collections.py

```diff
@@ -82,6 +83,7 @@ def upload_file(app, contents):
         resp_obj.raise_for_status()
         return file_uuid, resp_obj.json()["version"]
 
+    @eventually(timeout=60, interval=1, errors={requests.exceptions.HTTPError})
```

Comment:
Can this error be restricted further to `requests.codes.unprocessable`?

Reply:
I don't think it can with the eventually function (I can't think of a non-hacky way to add it).  It goes back to a custom retry since you need the response to get the code.  I'll make the change.